packages/editor-ui/src/plugins/i18n/locales/en.json

```diff
@@ -558,6 +559,7 @@
 	"executionsList.confirmMessage.message": "Are you sure that you want to delete the {numSelected} selected execution(s)?",
 	"executionsList.clearSelection": "Clear selection",
 	"executionsList.error": "Failed",
+	"executionsList.enqueued": "Enqueued",
```

Comment:
A user not on queue mode might be confused by the reference to an implementation detail. `Rate limited`? Might need product input.